prediction_market_agent/data_models/market_data_models.py

```diff
@@ -32,7 +44,7 @@ class OmenMarket(BaseModel):
     https://aiomen.eth.limo
     """
 
-    BET_AMOUNT_CURRENCY: t.ClassVar[str] = "xDai"
+    BET_AMOUNT_CURRENCY: Currency = Currency.xDai
```

Comment:
This can be kept as a class var, or the field can have lower case name. Right now, it seems like a constant, but it's changeable when creating the class.

Reply:
Ah good point!